cmd/crates/soroban-rpc/src/lib.rs

```diff
@@ -580,9 +581,17 @@ impl Client {
         tracing::trace!(?uri);
         Ok(Self {
             base_url: uri.to_string(),
+            timeout_in_secs: 30,
```

Comment:
this changes the default to 30 instead of 10, below, right? we should keep things consistent

Reply:
```suggestion
            timeout_in_secs: 10,
```